apis/r/tests/testthat/test-SOMASparseNDArray.R

```diff
@@ -309,3 +308,34 @@ test_that("platform_config defaults", {
 
   snda$close()
 })
+
+test_that("SOMASparseNDArray timestamped ops", {
+  uri <- withr::local_tempdir("soma-sparse-nd-array-timestamps")
+
+  ts <- function(s) as.POSIXct(s, origin="1970-01-01", tz="UTC")
+
+  # t=10: create 2x2 array and write 1 into top-left entry
+  ## NB: timestamp-on-write do not currently go through on write via tiledb-r
+  ##     this needs to be addressed first in tiledb-r, this package can then catch up
```

Comment:
Please include, as a code comment here, a link to the issue which tracks this to-do

Reply:
This actually is now in tiledb-r albeit not yet in the CRAN version.

Reply:
Thanks @johnkerl I cleaned up these comments for now but tracking it in broken-out #1450